### ISSUE TITLE
SpacesAroundOperators false-positives on dynamic funcrefs

### ISSUE THREAD

**Issue by glittershark** [rrrene/credo#464]
### Environment

* Credo version (`mix credo -v`): 0.8.6
* Erlang/Elixir version (`elixir -v`): 
Erlang/OTP 20 [erts-9.0.1] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:10] [hipe] [kernel-poll:false]

Elixir 1.5.2
* Operating system:
Linux 4.13.3-1-ARCH

### What were you trying to do?

The following code:
```elixir
map_fn = &(enum.__struct__).map/2
```

### Expected outcome

No warnings

### Actual outcome

"There are spaces around operators most of the time, but not here", specifically on the `/` operator.

**Comment by rrrene** [comment]
:+1: Thx for reporting this!

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍
